gslib/context_config.py

```diff
@@ -107,6 +110,58 @@ def _SplitPemIntoSections(contents, logger):
   return result
 
 
+def _check_path():
+  """Checks for context aware metadata. If it exists, returns the absolute path;
+  otherwise returns None.
+  Returns:
+      str: absolute path if exists and None otherwise.
+  """
+  metadata_path = os.path.expanduser(_CONTEXT_AWARE_METADATA_PATH)
+  if not os.path.exists(metadata_path):
+    _LOGGER.debug("%s is not found, skip client SSL authentication.", metadata_path)
+    return None
+  return metadata_path
+
+
+def _read_metadata_file(metadata_path):
+  """Loads context aware metadata from the given path.
+  Returns:
+      Dict[str, str]: The metadata.
+  Raises:
+      google.auth.CertProvisionError: If failed to parse metadata as JSON.
+  """
+  try:
+    with open(metadata_path) as f:
+      metadata = json.load(f)
+  except ValueError as caught_exc:
+    new_exc = CertProvisionError(caught_exc)
+    six.raise_from(new_exc, caught_exc)
+
+  return metadata
+
+
+def _default_command():
+  """Loads default cert provider command
```

Comment:
Docstring needs periods at the end of this line, the "str:" returns line, and the "CertProvisionError" line.